Web Inspector: REGRESSION(?): Sources: async boundary lines missing

dcrousso opened the pull request:
#### 52b00ad310456c02189426c1c439a3e2d2f6e763
<pre>
Web Inspector: REGRESSION(?): Sources: async boundary lines missing
<a href="https://bugs.webkit.org/show_bug.cgi?id=290026">https://bugs.webkit.org/show_bug.cgi?id=290026</a>

Reviewed by BJ Burg.

* Source/WebInspectorUI/UserInterface/Views/CallFrameTreeElement.css:
(.call-frame:is(.async-boundary, .truncated-boundary)):
The `::after` is sized `width: 100%`, which will always overflow, so ensure that `text-overflow` is not `ellipsis` (which comes from `.call-frame`/`WI.CallFrameView`).
Ideally we&apos;d change the size of the `::after`, but we don&apos;t know for sure the length of the async boundary name.

Canonical link: <a href="https://commits.webkit.org/292434@main">https://commits.webkit.org/292434@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fae31ad220db3e26d3783e1c89906d139203df06

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/95766 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/15376 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/5291 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/100819 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/46271 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/15663 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/23807 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/73025 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/30269 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/98769 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/11719 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/86493 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/53359 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/11446 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/4227 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/45610 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/81614 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/4335 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/102851 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/22825 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/16638 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/82061 "Found 1 new test failure: imported/w3c/web-platform-tests/css/css-view-transitions/active-view-transition-type-on-non-root.html (failure)") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/23077 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/82524 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/81426 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/20476 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/26001 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/3464 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/16161 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/22792 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/27941 "Built successfully") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/22451 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/25927 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/24193 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

burg reviewed (approved):
Huh, that's so strange to have regressed. Thanks for the fix!

dcrousso committed:
Web Inspector: REGRESSION(?): Sources: async boundary lines missing
https://bugs.webkit.org/show_bug.cgi?id=290026

Reviewed by BJ Burg.

* Source/WebInspectorUI/UserInterface/Views/CallFrameTreeElement.css:
(.call-frame:is(.async-boundary, .truncated-boundary)):
The `::after` is sized `width: 100%`, which will always overflow, so ensure that `text-overflow` is not `ellipsis` (which comes from `.call-frame`/`WI.CallFrameView`).
Ideally we'd change the size of the `::after`, but we don't know for sure the length of the async boundary name.

Canonical link: https://commits.webkit.org/292434@main
